crates/wasmtime/src/runtime/values.rs

```diff
@@ -1021,26 +1021,14 @@ mod tests {
     fn size_of_val() {
         // Try to keep tabs on the size of `Val` and make sure we don't grow its
         // size.
-        assert_eq!(
-            std::mem::size_of::<Val>(),
-            if cfg!(any(
-                target_arch = "x86_64",
-                target_arch = "aarch64",
-                target_arch = "riscv64",
-                target_arch = "s390x"
-            )) {
-                24
-            } else {
-                panic!("unsupported architecture")
-            }
-        );
+        assert!(std::mem::size_of::<Val>() <= 24);
     }
 
     #[test]
     fn size_of_ref() {
         // Try to keep tabs on the size of `Ref` and make sure we don't grow its
         // size.
-        assert_eq!(std::mem::size_of::<Ref>(), 24);
+        assert!(std::mem::size_of::<Ref>() <= 24);
```

Comment:
Can we make these precise for 64- and 32-bit archs? Because otherwise we might shrink it, forget to update these tests, and then regress the size again back to its current size.

Reply:
Good point yeah, I'll try. It's not quite as easy as 32-or-64 since the size differs between x86 and arm due to the alignment of u64 being different, but I'll probably just make these tests 64-bit only.